lib/api/src/js/imports.rs

```diff
@@ -130,7 +131,11 @@ impl Imports {
     /// Resolve and return a vector of imports in the order they are defined in the `module`'s source code.
     ///
     /// This means the returned `Vec<Extern>` might be a subset of the imports contained in `self`.
-    pub fn imports_for_module(&self, module: &Module) -> Result<Vec<Extern>, InstantiationError> {
+    pub fn imports_for_module(
+        &self,
+        _store: &mut impl AsStoreMut,
+        module: &Module
```

Comment:
Why do we need to nave a new `_store` param, if we are not using it?

Reply:
The store is needed here (for sys) - JS doesn't use it but I kept the interface consistent.
https://github.com/wasmerio/wasmer/blob/af5663beadb9c827841f52730bb90132bb37e910/lib/api/src/sys/imports.rs#L149

It's used for the situation that imported memory is needed (multithreading) - it will create the memory

Reply:
This is no longer needed and is removed from the new patch I made